src/js/post-911-gib-status/utils/helpers.js

```diff
@@ -19,6 +19,16 @@ export function formatVAFileNumber(n) {
   return number.replace(number.substring(0, lengthOfXString), `${'x'.repeat(lengthOfXString)}-`);
 }
 
+export function formatMonthDayFields(field) {
+  let displayValue;
+  if (field) {
+    displayValue = `${field.months || 0} months, ${field.days || 0} days`;
+  } else {
+    displayValue = 'unavailable';
```

Comment:
Let's check with @phiden on what this should say if the entitlement isn't supplied. None of the entitlement objects (total, used, remaining) are required, so it very well be that one is not defined, though it probably shouldn't happen in the wild (this is the problem I was hoping to address with this old ticket: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3336)

I think you're right that for any of these values that we can't rely in b/c they are not required fields in the API docs, we should supply a human-readable placeholder. Maybe we should also log a Sentry error so that we can monitor whether/how often this happens if at all.